File: LENGUAJES/C++/Calculadora.cpp
#include <iostream>

double Multiplicacion(double* a, double* b) {return *a * *b; }
double Resta(double* a, double* b) { return *a - *b; }
double Suma(double* a, double* b) { return *a + *b; }
double Division(double* a, double* b) { return (*b != 0 ? *a / *b : 0.0); }

int main(){
    
    double Num1, Num2, Resultado;
    char operador;

    std::cout << "ingrese el primer numero..."; std::cin>>Num1;
    std::cout << "ingrese el operador..."; std::cin>>operador;
    std::cout << "ingrese el segundo numero..."; std::cin>>Num2;

    switch( operador ) {
        case '*' : 
            Resultado = Multiplicacion(&Num1, &Num2);
            break;
        case '-' :
            Resultado = Resta(&Num1, &Num2);
            break;
        case '+' :
            Resultado = Suma(&Num1, &Num2);
            break;
        case '/' :
            Resultado = Division(&Num1, &Num2);
            break;
        default :
            std:: cout << "el operador es invalido...";
            break;
    }

    std::cout << "El resultado de la operacion es = " << Resultado << "\n";

    return 0;
}

File: LENGUAJES/Python/Calculadora.py
import os
import time
def Multiplicacion(a, b):
    return a * b

def Sumar(a, b):
    return a + b

def Restar(a, b):
    return a - b

def Division(a, b):
    return a / b if b != 0 else  "No se puede dividir entre cero"

Numero1 = float(input(" Ingrese el primer numero..."))
Operador = input("Ingrese el operador...")
Numero2 = float(input("Ingrese el segundo numero..."))
Resultado = 0

if (Operador == "*"):
    Resultado = Multiplicacion(Numero1, Numero2)
elif (Operador == "+"):
    Resultado = Sumar(Numero1, Numero2)
elif (Operador == "-"):
    Resultado = Restar(Numero1, Numero2)
elif (Operador == "/"):
    Resultado = Division(Numero1, Numero2)
else:
    print("ingreso un operador invalido...")

print(f"El resultado es: {Resultado}")